instrumentation/grpc-1.5/library/src/main/java/io/opentelemetry/instrumentation/grpc/v1_5/GrpcTracingBuilder.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.grpc.v1_5;
+
+import io.opentelemetry.api.OpenTelemetry;
+
+/** A builder of {@link GrpcTracing}. */
+public final class GrpcTracingBuilder {
```

Comment:
Note, I proactively went with a builder for this single property here because unlike SDK, where we generally proactively define functionality, with instrumentation it's more common to be reactive and require different settings.